Deprecate PCI DSS for Logs and APM (#30919)
DataDog/documentation: content/en/data_security/pci_compliance.md

---
title: PCI DSS Compliance
further_reading:
- link: "https://trust.datadoghq.com/"
  tag: "Datadog Trust Center"
  text: "Learn about Datadog's security posture and review security documentation"
---

## Overview

The Payment Card Industry (PCI) Data Security Standard (DSS) has rigorous monitoring and data security requirements for all merchants, service providers, and financial institutions. To meet these requirements, organizations often separate PCI-regulated data (such as cardholder data) and non-regulated data into different applications for monitoring and compliance purposes.

> **Datadog's tools and policies comply with PCI v4.0**. To understand the full scope of Datadog's environment and how it relates to customer responsibilities under the relevant PCI-DSS controls, download the Customer Responsibility Matrix and the Attestation of Compliance (AoC) from the [Datadog Trust Center][1].

Author: @jinatdatadog Can you confirm these docs? I couldn't find them.

Datadog's Attestation of Compliance (AoC) reflects the tools and policies we have in place to maintain a Connected PCI environment as a service provider. The Datadog platform supports connections to cardholder data environments (CDE) as a Connected PCI environment, but does not serve as a CDE itself for storing, processing, or transmitting cardholder data (CHD).

> It is the customer's responsibility to prevent any CHD from entering the Datadog platform. See below for some tools customers can use to meet this requirement.

Author: ```suggestion
It is your responsibility to prevent any CHD from entering the Datadog platform. 
```
